### ISSUE TITLE
Long Width PETSCII Aspect Ratio Locks and makes zooming/padding to a thin window

### ISSUE THREAD

**Issue by wbochar** [nurpax/petmate#193]
Create a PETSCII Image that is 300x25
The display shows a fully zoomed out image, zoom in and it will only zoom in the size ratio of the original window.
if would be nice to be able to set the viewport/zoom ratio.

**Comment by hakkanpersson** [comment]
Agree. Yes, this would truly simplyfy things when working on either wide or tall petscii screens.
